go.mod

```diff
@@ -20,6 +20,6 @@ require (
 	go.starlark.net v0.0.0-20190702223751-32f345186213
 	golang.org/x/arch v0.0.0-20190927153633-4e8777c89be4
 	golang.org/x/sys v0.0.0-20190626221950-04f50cda93cb
-	golang.org/x/tools v0.0.0-20191127201027-ecd32218bd7f
+	golang.org/x/tools v0.0.0-20200415034506-5d8e1897c761
```

Comment:
I don't think this needs an updated version of golang.org/x/tools so this should be reverted. If I'm wrong and it does the vendor directory should also be updated.

Reply:
Yeah, for sure it is not necessary. My bad, I think I update this while doing the rebase of the branch for some reason.